presto-orc/src/test/java/com/facebook/presto/orc/OrcTester.java

```diff
@@ -935,6 +928,15 @@ public static void assertFileContentsPresto(
         assertEquals(rowsProcessed, expectedValues.get(0).size());
     }
 
+    static void assertBlockEquals(Type type, Block block, List<?> expectedValues, int offset)
+    {
+        int positionCount = block.getPositionCount();
+        List<Object> data = new ArrayList<>(positionCount);
```

Comment:
this variable is not used; remove

Reply:
@mbasmanova Variable removed and tests passed. Thank you! Merry Christmas!